CHANGES.rst

```diff
@@ -20,6 +20,8 @@ releases are available on `Anaconda.org <https://anaconda.org/gettsim/gettsim>`_
 * Child tax allowance modelled as two separate items. (:ghuser:`Eric-Sommer`)
 * Alimony advance payment (*Unterhaltsvorschuss*) now modelled more in line
   with the law (:ghuser:`Eric-Sommer`)
+* Implement *Familienentlastungsgesetz* on income tax tariff and child tax
```

Comment:
This is only Artikel 3 of FamEntlastG, right? Maybe just say which functions are affected.